src/engine/server/sv_client.cpp

```diff
@@ -658,20 +662,20 @@ when the cvar is set to something, the download server will effectively never us
 */
 static bool SV_CheckFallbackURL( client_t *cl, const char* pakName, int downloadSize, msg_t *msg )
 {
-	if ( !sv_wwwFallbackURL->string || !sv_wwwFallbackURL->string[0] )
+	if ( sv_wwwFallbackURL.Get().empty() )
 	{
 		return false;
 	}
 
-	Log::Notice( "clientDownload: sending client '%s' to fallback URL '%s'\n", cl->name, sv_wwwFallbackURL->string );
+	Log::Notice( "clientDownload: sending client '%s' to fallback URL '%s'\n", cl->name, sv_wwwFallbackURL.Get().c_str() );
```

Comment:
I think you don't need the .c_str()
```suggestion
	Log::Notice( "clientDownload: sending client '%s' to fallback URL '%s'\n", cl->name, sv_wwwFallbackURL.Get() );
```